### ISSUE TITLE
Target temperature shows NaN with 0.x step size

### ISSUE THREAD

**Issue by ThomDietrich** [artem-sedykh/mini-climate-card#97]
```yaml
type: custom:mini-climate
entity: climate.room
target_temperature:
  step: 1.0
```

![image](https://user-images.githubusercontent.com/2870104/192597439-f488af26-374f-447c-9b78-b09fc4841b1c.png)

```yaml
type: custom:mini-climate
entity: climate.room
target_temperature:
  step: 0.5
```

![image](https://user-images.githubusercontent.com/2870104/192597761-2b63de3f-ec84-4c4b-9e81-55490f72359e.png)

Relates to https://github.com/artem-sedykh/mini-climate-card/pull/50
CC @GuyLewin

**Comment by regevbr** [comment]
fixed in 2.3.1

**Comment by ThomDietrich** [comment]
Working as expected. Thanks a lot!!
